### PR TITLE
Add fsOpenBisStorage and fsStorageGetSize

### PR DESCRIPTION
**Pull request by tesnos**

### CODE REVIEW

File: nx/source/services/fs.c [hunk 1]
```diff
@@ -1106,6 +1143,39 @@ Result fsStorageRead(FsStorage* s, u64 off, void* buf, size_t len) {
     return rc;
 }
 
+Result fsStorageGetSize(FsStorage* f, u64* out) {
```
Comment (yellows8) [inline nx/source/services/fs.c]: FsStorage* s (likewise for .h)

Comment (tesnos) [reply nx/source/services/fs.c]: changed

File: nx/include/switch/services/fs.h [hunk 2]
```diff
@@ -144,6 +144,7 @@ void fsExit(void);
 
 Service* fsGetServiceSession(void);
 
+Result fsOpenBisStorage(FsStorage* out, u32 PartitionId);
 /// Do not call this directly, see fs_dev.h.
```
Comment (yellows8) [inline nx/include/switch/services/fs.h]: add newline

File: nx/include/switch/services/fs.h [hunk 3]
```diff
@@ -144,6 +144,7 @@ void fsExit(void);
 
 Service* fsGetServiceSession(void);
 
+
```
Comment (yellows8) [inline nx/include/switch/services/fs.h]: Wrong line.

**Pull request merged**